packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -263,7 +276,17 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             assert(batch[batch.length - 1].metadata?.batch === false, 0x3cb /* no end batch markup */);
         }
 
-        this.connectionManager.sendMessages(batch);
+        // TODO: revisit
+        if (batch.length >= 2) {
+            this.connectionManager.sendMessages([{
+                type: GroupedBatchOpType,
+                clientSequenceNumber: batch[0].clientSequenceNumber, // TODO: doesn't matter what we put here because we forget this message
+                referenceSequenceNumber: batch[0].referenceSequenceNumber, // TODO: doesn't matter what we put here because we forget this message
```

Comment:
Somewhat similar to above: While loader assigns it (and this number is not visible to local client directly), it needs to be "right" (from runtime POV) in a sense of properly referring to a state of container at time this op was made. I.e. it needs to be virtualized

Reply:
My thinking was this new op is essentially ignored once DeltaManager gets the ack for it. It recognizes it and decomposes to the original ops with virtual sequence numbers (which contain the virtualized reference sequence number from when it was submitted).